cincinnati/src/plugins/mod.rs

```diff
@@ -61,15 +61,15 @@ where
 
 /// Trait to be implemented by internal plugins with their native IO type
 pub trait InternalPlugin {
-    fn run_internal(&self, InternalIO) -> Fallible<InternalIO>;
+    fn run_internal(&self, _: InternalIO) -> Fallible<InternalIO>;
 }
 
 /// Trait to be implemented by external plugins with its native IO type
 ///
 /// There's a gotcha in that this type can't be used to access the information
 /// directly as it's merely bytes.
 pub trait ExternalPlugin {
-    fn run_external(&self, ExternalIO) -> Fallible<ExternalIO>;
+    fn run_external(&self, _: ExternalIO) -> Fallible<ExternalIO>;
```

Comment:
see my previous comment.

```suggestion
    fn run_external(&self, input: ExternalIO) -> Fallible<ExternalIO>;
```